### PR TITLE
[Sprint4][Team3][Anthony]-Implement Headless Chrome Test for React UI for /users

### PR DESCRIPTION
**Pull request by AnthonySDi**
Link to [my issues](https://github.com/North-Seattle-College/ad440-winter2021-tuesday-repo/issues/259)

### CODE REVIEW

**Review by elb-dev — APPROVED** [review]
Looks good!

File: testing/ui-tests/ui-test-users.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const puppeteer = require('puppeteer');
+
+(async () => {
+  const browser = await puppeteer.launch();
```
Comment (OyunominJuno) [inline testing/ui-tests/ui-test-users.js:4]: you can add an argument "{headless: true}" inside the launch method to make sure it is headless

File: testing/serverless-artillery/serverless-artillery-users-id-tasks.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 config:
   target: "https://nsc-func-dev-usw2-tuesday.azurewebsites.net"
   phases:
-    - duration: 300
+    - duration: 3600
```
Comment (OyunominJuno) [inline testing/serverless-artillery/serverless-artillery-users-id-tasks.yml]: would this make it 60 mins? wouldn't that be super long?

**Review by OyunominJuno — APPROVED** [review]
good job! please consider the comments, otherwise they are working scripts

**Review by sunshinejen — APPROVED** [review]

**Review by sunshinejen — APPROVED** [review]
Looks good!

**Pull request merged**